students/ptfbanks/Lesson03/3-2_list_lab.py

```diff
@@ -0,0 +1,64 @@
+print ("Today we are going to be playing with fruits.  It's a healthy exercise.  Don't take it personally")
```

Comment:
print() is a function, so you can remove the space between print and the parentheses.